src/common/vordr.ts

```diff
@@ -0,0 +1,92 @@
+import { isInSubnet, isIP } from 'is-in-subnet';
+import { Context } from '../Context';
+import { Comment, User } from '../entity';
+import { logger } from '../logger';
+import { counters } from '../telemetry';
+import { Brackets } from 'typeorm';
+
+const vordrIPs =
+  process.env.VORDR_IPS?.split(',').filter((ip) => Boolean(ip)) || [];
+
+const vordrWords =
+  process.env.VORDR_WORDS?.split(',').filter((word) => Boolean(word)) || [];
```

Comment:
we can lowercase the words here instead of lowering them in real-time. Nice to have

Reply:
Good idea. Added.